emails/views.py

```diff
@@ -663,6 +673,12 @@ def _handle_bounce(message_json):
             ),
             1
         )
+        # if an email bounced as spam, set to auto block spam for this user
+        # and DON'T set them into bounce pause state
+        if (any('spam' in val.lower() for val in recipient.values())):
+            profile.auto_block_spam = True
+            profile.save()
+            continue
```

Comment:
We record bounce on the user attribute `last_soft_bounce` if it’s user’s email server responding that it is not receiving emails right now (bounced). But when we get a bounce from the email server with the reason being that we are relaying spam, we instead record `auto_block_spam`. This will help us continue to relay emails reliably without being marked as a spammer or putting users relay account on pause.